### PR TITLE
Converge to a single Jenkins configuration for all jobs.

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

File: docs/development/ci_images.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# CI Images
+
+This page documents how the CI images for Shaken Fist are setup in our testing cloud account. At the moment Shaken Fist CI is not self hosting, and runs on Google Compute Engine. It would be good to change this in the future.
+
+## The sf-privci image
+
+Our test nodes are used for a single test each, and are based on the "sf-privci" series of images in the relevant Google Cloud project. These images need regular updates or the dist-upgrade time on startup gets our of hand. The process to do that is as follows:
```
Comment (mandoonandy) [inline docs/development/ci_images.md:7]: `out of hand` not `our`

**Review by mandoonandy — APPROVED** [review]
LGTM

**Comment by mikalstill** [comment]
The "all test runs manual" bit is required for the nightlies to work.

**Review by mandoonandy — APPROVED** [review]
LGTM

**Pull request merged**